opm/input/eclipse/Schedule/Schedule.hpp

```diff
@@ -87,6 +87,7 @@ namespace Opm
         std::optional<int> output_interval;
         double sumthin{-1.0};
         bool rptonly{false};
+        bool gaslift_opt{false};
```

Comment:
If we're going to use a `bool` can we please name it something more descriptive than `gaslift_opt`&mdash;e.g., `alq_is_gaslift` or similar?

Reply:
Sure - can use a better name; although the good name for this member is in my opinion something like `gaslift_optimization_is_active`  - which is very long. That undefined ALQ is converted to GRAT (the whole point of the excercise) - is not known/relevant at this particular location.

Reply:
How about `gaslift_opt_active`?

Reply:
Have changed to `gaslift_opt_active`